### PR TITLE
Basic Webpage layout

### PR DESCRIPTION
**Pull request by Sophia-bhs**
Hey team, please test and review.

### CODE REVIEW

**Comment by gazzyfire** [comment]
index.php is missing a </form> at the end
added a placeholder name for email and password for index.php
i've added some code styling and webpage format (background colour and center format) to the index.php login form
we can remove login.html if we aren't using that anymore
Otherwise, rest is fine to pull

File: index.php [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    <body bgcolor="#E7FBFC">
+        <div>
+            <h2>
+                Login
+            </h2>
+        </div>
+        
+        <form action="login.php" method="POST">
+            <div class="container">
+                <label><b>Email</b></lable>
```
Comment (nisargkanthi) [inline index.php]: label is spelled wrong

**Review by nisargkanthi — COMMENTED** [review]
It is really well done , except for some spelling mistakes

**Review by gazzyfire — APPROVED** [review]
I also changed submit to a button in line 27 of index.php

File: index.php (hunk 1 above)
Comment (gazzyfire) [reply index.php]: i just changed it HAHAHA, spotted it at the same time

**Review by nisargkanthi — APPROVED** [review]
It's really well done, except  for a few spelling mistakes.

**Review by z5256089 — APPROVED** [review]

**Pull request merged**